pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -199,12 +200,12 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 		allErrors = append(allErrors, err)
 	}
 	allErrors = append(allErrors, metrics.ValidateShowHiddenMetricsVersion(kc.ShowHiddenMetricsForVersion)...)
-
-	logOption := logs.NewOptions()
-	if kc.Logging.Format != "" {
-		logOption.LogFormat = kc.Logging.Format
+	if kc.Logging.Format == "" {
+		kc.Logging.Format = logs.DefaultLogFormat
+	}
```

Comment:
Don't mutate kubelet configuration inside validation. This belongs in the defaulting function.